Sources/RealDeviceMapLib/Controller/DBController.swift

```diff
@@ -336,7 +336,7 @@ public class DBController {
 
             // Schema
             //  swiftlint:disable:next line_length
-            let commandSchema = Shell("bash", "-c", mysqldumpCommand + " --column-statistics=0 --set-gtid-purged=OFF --skip-triggers --add-drop-table --skip-routines --no-data \(self.database) \(tablesSchema) -h \(self.host) -P \(self.port) -u \(self.rootUsername) -p\(self.rootPassword.stringByReplacing(string: "\"", withString: "\\\"")) > \(backupFileSchema.path)")
```

Comment:
why did you changed backup scripts?!
are you running mysql 5.7 or 8?

Reply:
had to modify, mariadb 10.6.10 didn't like it at all.

Reply:
so we should fix it for both.. because when you remove it it will crash on mysql ;)

Reply:
finally back to work on things.  i don't know how to make it compatible.

Reply:
did find this in perfect mysql, need to see what data it gives back, if anything like the version from shell one should be able to check and take action based on such.

public static func clientInfo() -> String